scheduler-server/src/test/java/gov/nasa/jpl/aerie/scheduler/server/services/SchedulingDSLCompilationServiceTests.java

```diff
@@ -44,7 +47,7 @@ public PlanMetadata getPlanMetadata(final PlanId planId) {
       }
 
       @Override
-      public Plan getPlanActivities(final PlanMetadata planMetadata, final Problem mission)
+      public Triple<Plan, MerlinPlan, Map<SchedulingActivityInstanceId, ActivityInstanceId>> getPlanActivities(final PlanMetadata planMetadata, final Problem mission)
```

Comment:
(not a review) What's the difference between a `Plan` and a `MerlinPlan`?

Reply:
A `Plan` is the scheduler's representation of a plan, with partially instanciated activity parameters, activity durations and so on.  A `MerlinPlan` is limited to the representation of a plan in the merlin database.